pkg/envoy/rds/route/route_config.go

```diff
@@ -206,14 +207,14 @@ func buildEgressRoutes(routingRules []*trafficpolicy.EgressHTTPRoutingRule) []*x
 		// Build the route for the given egress routing rule and method
 		// Each HTTP method corresponds to a separate route
 		for _, httpMethod := range allowedHTTPMethods {
-			route := buildRoute(rule.Route.HTTPRouteMatch.PathMatchType, rule.Route.HTTPRouteMatch.Path, httpMethod, nil, rule.Route.WeightedClusters, rule.Route.TotalClustersWeight(), outboundRoute)
+			route := buildRoute(rule.Route.HTTPRouteMatch.PathMatchType, rule.Route.HTTPRouteMatch.Path, httpMethod, nil, rule.Route.WeightedClusters, rule.Route.TotalClustersWeight(), outboundRoute, rule.Route.RetryPolicy)
```

Comment:
It's better to simply pass `rule.Route` instead of so many of its sub fields to the function. But that can be an improvement addressed later.

Reply:
https://github.com/openservicemesh/osm/issues/4020